zerver/openapi/zulip.yaml

```diff
@@ -9767,9 +9768,6 @@ paths:
 
                           **Changes**: The emoji parameters are new in Zulip 5.0 (feature level 86).
                           Previously, Zulip did not support emoji associated with statuses.
-
-                          A status that does not have an emoji associated with it is encoded
-                          with `emoji_name=""`.
```

Comment:
What's the thikning behind removing this line?

Reply:
Oh, will these fields not appear in the `/register` response at all if there's no associated emoji?

Reply:
Exactly, the `/register` response only fills in the fields for a user's status if they have associated value. So there will never be an empty string. The field will just not exist in the object. So I removed it here because it felt confusing.

Reply:
I guess one thing I notice now is that the `/update-status` endpoint does not have the **Changes** note for the addition of emojis to statuses. Follow-up task or should I go ahead and incorporate that in this update?

I think in my mind, this PR was an immediate step to fix the clear bugs with the documentation: remove `EmojiBase`, take steps towards clarifying the object fields as being present / not-present vs the incorrect implication that empty strings would be returned.

Reply:
Yeah, https://zulip.com/api/update-status lacking a changes note is a clear bug.